src/admin-views/settings/featured/container.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Featured settings box.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this              Template object.
+ * @var string                       $title             Featured settings title.
+ * @var string                       $description       Featured settings description/HTML.
+ * @var string                       $content_template  Template used for the content section.
+ * @var array                        $content_context   Context for template used for content.
+ * @var string[]                     $classes           Array of classes.
+ * @var array                        $links             Array of arrays for links.
+ */
+
+$default_classes = ['tec-tickets__admin-settings-featured'];
```

Comment:
Not sure if we use the concept of two arrays for managing CSS classes somewhere else. Maybe we can just rely on the $classes var, and assign `$classes[] = 'tec-tickets__admin-settings-featured';` so that we avoid doing the array merge?